### ISSUE TITLE
NoMethodError: undefined method `loopback?' for 127.0.0.1:IPAddress::IPv4

### ISSUE THREAD

**Issue by tetherit** [ipaddress-gem/ipaddress#25]
I'm trying to check if my IP is a loopback ip, I get this:

``` ruby
ip = IPAddress('127.0.0.1').loopback?
NoMethodError: undefined method `loopback?' for 127.0.0.1:IPAddress::IPv4
    from (irb):9
    from /usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/gems/1.9.1/gems/railties-3.2.8/lib/rails/commands/console.rb:47:in `start'
    from /usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/gems/1.9.1/gems/railties-3.2.8/lib/rails/commands/console.rb:8:in `start'
    from /usr/local/Cellar/ruby/1.9.3-p194/lib/ruby/gems/1.9.1/gems/railties-3.2.8/lib/rails/commands.rb:41:in `<top (required)>'
    from script/rails:6:in `require'
    from script/rails:6:in `<main>'
```

Any ideas?

**Comment by tisba** [comment]
My current workaround is to check if a given IP address is included in `127.0.0.0/8` (I only need v4 currently). Would be a nice addition to the gem's feature set!

**Comment by francisluong** [comment]
It'd be easy enough to include this in a PR but when is 1.0 coming out.  I would imagine most are hesitant to contribute changes that would get clobbered by a major release.

**Comment by bluemonk** [comment]
This was fixed by Mike in [f562a7f7](https://github.com/bluemonk/ipaddress/commit/f562a7f7a032b381cdc8b897ec50ed220bcc9452)

**Comment by sandstrom** [comment]
I'm doing some issue gardening 🌱🌿 🌷 and came upon this issue. 

Fixed as per the comment above. Closing this issue, to avoid having lots of old issues open.
